2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_130912-3909nd2z/logs/debug.log
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_130912-3909nd2z/logs/debug-internal.log
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_init.py:init():404] calling init triggers
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 13:09:12,050 INFO    MainThread:621067 [wandb_init.py:init():460] starting backend
2022-02-28 13:09:12,050 INFO    MainThread:621067 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 13:09:12,056 INFO    MainThread:621067 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 13:09:12,060 INFO    MainThread:621067 [backend.py:ensure_launched():222] started backend process with pid: 621076
2022-02-28 13:09:12,060 INFO    MainThread:621067 [wandb_init.py:init():469] backend started and connected
2022-02-28 13:09:12,062 INFO    MainThread:621067 [wandb_init.py:init():533] updated telemetry
2022-02-28 13:09:12,063 INFO    MainThread:621067 [wandb_init.py:init():563] communicating current version
2022-02-28 13:09:12,523 INFO    MainThread:621067 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 13:09:12,523 INFO    MainThread:621067 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 13:09:13,090 INFO    MainThread:621067 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 13:09:15,734 INFO    MainThread:621067 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 13:09:15,734 INFO    MainThread:621067 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 13:09:15,735 INFO    MainThread:621067 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 13:09:15,736 INFO    MainThread:621067 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 13:09:15,736 INFO    MainThread:621067 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 13:09:31,456 INFO    MainThread:621067 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-28 13:09:31,457 INFO    MainThread:621067 [wandb_run.py:_restore():1752] restore
2022-02-28 13:09:34,267 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 13:09:34,564 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 13:09:35,108 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 6860
}

2022-02-28 13:09:35,211 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16459
}

2022-02-28 13:09:35,313 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16459
}

2022-02-28 13:09:35,415 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16459
}

2022-02-28 13:09:35,516 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 16459
}

2022-02-28 13:09:35,618 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:35,720 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:35,823 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:35,925 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:36,028 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:36,518 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}

2022-02-28 13:09:37,326 INFO    MainThread:621067 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 16459
  total_bytes: 16459
}
local_info {
}

2022-02-28 13:09:38,390 INFO    MainThread:621067 [wandb_run.py:_append_history():2130] rendering history
2022-02-28 13:09:38,391 INFO    MainThread:621067 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-28 13:09:38,391 INFO    MainThread:621067 [wandb_run.py:_append_files():2180] logging synced files
